unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore_ExploreV2.cs

```diff
@@ -11,6 +11,7 @@ public class DataStore_ExploreV2
         public readonly BaseVariable<int> currentSectionIndex = new BaseVariable<int>(0);
         public readonly BaseVariable<bool> placesAndEventsVisible = new BaseVariable<bool>(false);
         public readonly BaseVariable<bool> profileCardIsOpen = new BaseVariable<bool>(false);
+        public readonly BaseVariable<bool> isSomeModalOpen = new BaseVariable<bool>(false);
```

Comment:
I find the name of this variable a bit strange/confusing, can there be 2 modals open at the same time? If that's the case then maybe it's better to have a `List` of open modals here and then check the `Count()` for that list instead of this bool. Otherwise if you have 2 open modals and you close the 2nd, it would override this bool setting it as false when in reality there's still another modal open.

If only 1 modal can be opened at a time then I think it's fine as it is.

Reply:
Yeah, I understand what go you mean. At the moment and inside the StartMenu's scenario we cannot have more than 1 modal open at the same time (at least we have not had the necessity yet), but if we would need it in the future then we would convert this variable to something more complex like you suggest. I'll take it into account, thanks for the feedback!